package.json

```diff
@@ -254,6 +254,10 @@
                     "command": "extension.vsKubernetesConfigureFromCluster",
                     "when": "view == extension.vsKubernetesExplorer"
                 },
+                {
+                    "command": "extension.vsKubernetesUseKubeconfig",
+                    "when": "view == extension.vsKubernetesExplorer"
```

Comment:
Let's put a separator between this and the Add/Create Cluster commands (add `"group": 0` to the cluster commands and group 1 to this one).

Reply:
done.